### ISSUE TITLE
Ensuring LAND is put to good use and not just held or abandoned 

### ISSUE THREAD

**Issue by dwrowley** [decentraland/proposals#8]
Tier 2 Proposal

**Project:** Land Use Fees

**Description:** In order to encourage LAND owners to not just let plots lie fallow and held indefinitely, I propose a periodic (annual?) land use fee be levied.  This encourages LAND owners to develop the property in order to generate enough income to at least cover the land use fees.   This also has the advantage that if an owner abandons the plot or loses access to it, that the LAND can be reclaimed and put to better use.

I propose the land use fee be initially set at 10 MANA per year (1% of the purchase price), with the actual rate to be decided by / updated by the DAO over time.

If the fee is not paid (by the anniversary of the purchase date of the parcel), then the LAND would be put up for auction (over a period of say 1 week) where the highest bid would win ownership of that plot.   Multiple adjacent plots available at the same time could be put up for auction as a block in order to facilitate purchases of larger regions.

As the value of LAND grows, and a resale market is generated, instead of using the 10 MANA figure, it could move to more of an appraised value model (where the land owner would be charged 1% of the appraised value rather than 1% of the initial purchase price).  Appraised value could be determined by the price of the property when it was last sold, or potentially based on the traffic (# visitors over time) that the LAND generated.  For a traffic-based model, the formula could be to take the overall economic value of Decentraland (GDP) and calculate that LAND's expected pro-rata share of that value based on the percent of traffic that lot had of the overall world.  Alternatively the value of the LAND could be decided by a prediction market.   The floor of course could continue to be set at 10 MANA.

Proceeds from both land use fees and abandoned lots being auctioned could be placed into a separate fund, managed by the DAO, that could be used for community focused projects (setting up park areas, community services, or other community-minded projects).   Proposals could be submitted to the DAO for consideration as to how these funds could best be used for specific projects.

In order to ensure land owners are fully aware of the need to pay the land use fee and the consequences of not doing so, there should be some mechanism to inform / alert the land owner of the appropriate deadlines, the mechanism for making the payment, confirmation that payment was received and that the LAND was in good standing, etc.

With these principles in place, owners will have a good ongoing incentive to continually ensure the LAND they own is being put to good use and serves the interest of the overall community.

**Comment by codetheweb** [comment]
This is a powerful idea.  It potentially could work even better with a simple upvote/downvote system.

IMHO, one of the biggest problems Decentraland faces is avoiding two downfalls: becoming a trolling graveyard, or, equally devastating; a barren wasteland with no content.  These are not especially easy problems to solve (especially for a decentralized project).  Getting rid of trolls usually involves moderators (a point of centralization).  And people need incentives to create good content.

One way to migrate these two problems could be a combination of a voting system and your "land use fees" (which I'll just call taxes 😛).  Every LAND's vote counter starts at 0 (an upvote increases the counter by 1, a downvote decreases it by 1).  At 0, the owner doesn't have to pay any tax on their land.  However, if the LAND's vote counter goes below 0, the owner must pay taxes in proportion to `abs(votes)` (traffic would also be a good metric to include in determining the amount to tax so that downvotes and upvotes would affect a heavily-traveled sector of Decentraland more than places most users never visit).  In practice, this means that users can vote structures on a plot of land out of existence.
On the flip side, what happens if a LAND's vote counter goes positive?  Well, MANA is starting to roll in from the land taxes, let's put that to good use.  After a certain period of time (say 48 hours), MANA from the land taxes is distributed to owners of upvoted LAND, in proportion to the number of votes.

This introduces both negative and positive economic incentives for the community to create and keep great content.

**Comment by ZCoderGuy** [comment]
I think a good way to do it would be a system where after a certain amount of time the land is closed to the owner developing on it or writing scripts for it and is notified that it is locked down. If they confirm that they still intend on using it or if they add to the land in some way, it is still theirs. If they do not affirm that they want to do something on it, it is put up for auction. If the owner wants the land at this point they could buy the land back at half the current price. Of course in this scenario, there is no community input, and the specific time constraints would need to be decided, but I think this is fair.

**Comment by pookage** [comment]
I actually think that @ZCoderGuy has the right solution in this case. @dwrowley's original solution would cause problems for new users keeping hold of their land if they are unable to acquire MANA, and benefit long-term users more likely to be sitting on piles of MANA from the initial buzz.  Even 1000MANA would leave a plot unusable for 100 years! 

@codetheweb's solution is vulnerable to coordinated trolling - particularly if a user goes against the hivemind, with the consequences of resisting trolling being financial; only acting as an incentive for further trolling later on.

Having a simple arrive after a few months or a year of inactivity on your account, asking if you'd still like to keep your land, would weed out the people who don't care due to the fact that they'd just ignore the email - leaving the land to default to open sale / auction.

Neat, tidy, and less open to abuse I rex.

**Comment by cyberworm1** [comment]
I think the issue I have with these options, is that reclamation of land via auction is a tough pill to swallow. If I'm not mistaken, part of the beauty of DCL is persistent ownership. In order to take land, there has to be a transaction from the owner to the receiver. Secondly, if the land is put up for auction, the owner of the land should be entitled to at least recoup their base investment in the property.

**Comment by dwrowley** [comment]
I understand the concerns - but in a fully decentralized environment what if a given landowner decides to completely abandon their land? (dies, loses interest, etc.) - should the land never be reclaimed?

**Comment by cyberworm1** [comment]
That is definitely an interesting question. Someone had mentioned a dead swap, though what that involves or how it works is beyond my knowledge.  I think you've got a good idea, but just taking something that someone paid money for is a hard pill to swallow, especially in a situation that could have entirely anonymous owners that may be "active" but with no real way to be notified.  In a sense, it's paramount to asking registrars to pull domain names, just because they are parked. Though in that scenario, someone obviously has to pay once a year for maintenance. 

It's a tough call. I know of other virtual platforms that have had to deal with it, and I think in the end the communities have figured out ways to work around dead plots without having to resort to repossession or imminent domain.

**Comment by pookage** [comment]
I think I have the solution! We're solving the wrong problem - we're attempting to change ownership of abandoned land, and provide incentives for users to maintain the land that they do have, right? 

So, what about if we introduce a 'nature' mechanic to the system? Untouched land grows long grass, and vines crawl up the surfaces of abandoned buildings. We make abandoned plots LOOK abandoned. It could just be as simple as applying a second transparent material to the mesh, which would be straightforward! On a more complex scale we could have a real jungle vibe going. That way even abandoned plots wouldn't look like a mish-mash of half-made experiments or empty plains, but part of a larger untamed wild. 

Thoughts?

**Comment by codetheweb** [comment]
^ Good idea.  That would be a lot more interesting than a brown plain as far as you can see.

However, it still doesn't solve the core issue - how can land that is not being put to good use (either abandoned or trolled) be automatically 'put back in the system'?

I like your idea for something like a transition period during which the system decides what to do with the land, but I don't think it would work as a permanent solution.  Do you really want a square chunk of LAND in the middle of a bustling city to be a jungle?

**Comment by ZCoderGuy** [comment]
@pookage I like your idea of making the land LOOK unused, but that still doesn't solve the problem of it being unused. The way I understand, and please correct me if I'm wrong, there is a finite amount of land. If all of the land is used up even 1/15 or 1/16 parcels of land isn't being actively developed upon, that could still add up to be a lot of land that is unused. Say there are 500,000 parcels of land and 1/20 parcels of land aren't being used. That means that 25,000 parcels of land aren't being used(I think my math may be off, just napkin math). After writing all of this and thinking about it a little more, I have come up with an addition to my original idea. What if the whole world had a section of land reserved for unused pieces of land. Once your land was unused for a specific length of time the system would move the area of land you owned into the unused section. After you followed the procedure of my above suggestion, or someone else's, the land could be moved back into civilization.

**Comment by pookage** [comment]
Ah, my understanding was that there's not a finite amount of LAND - as long as users are able to buy MANA, they can generate new land with it. The amount of MANA available will be limited, true, but it'll always be growing - so there wouldn't be a situation in which the only way to get land was to take it from someone else. 

My thoughts on the literal jungle rather than the concrete one were that adjacent abandoned plots would then form a whole together, and that lone plots would become more like unique public gardens than anything else. 

In terms of implementation, it would be fairly straightforward. In A-Frame, a new scene is entirely empty - you need a add a floor etc in. So rather than adding in `<a-plane>` we could create our own custom component `<a-land>` that would track how long since it was last used, and start spawning grass, vines, trees etc. 

I'm working on some plant components at the moment, so will expand that to look into growing around objects later on. 

Someone posted a proposal earlier that could be the missing piece of the puzzle - (wormhole adjacency)[https://github.com/decentraland/proposals/issues/27]  - after an amount of time abandoned, instead of recycling or deleting the LAND, we could just de-reference it, and stitch its adjacent LAND together with a wormhole? Might cause folding issues, I guess, but it's another potential solution.

**Comment by cyberworm1** [comment]
I like the idea of having the land look a bit "unkempt" in some ways.  What that should look like is probably up for debate. I'm not sure I've even heard of or seen what the base aestethic is going to be from the start. 

I think in this proposal what I take issue with, is the idea that because someone may not like what someone is doing with a plot of land (and yes, perhaps they are just sitting on it for some reason or another), that it should be ok to take it away from them. 

Honestly, I plan on having a few pieces of land and I can promise you that they won't all be used, because having some open space would be aestetically pleasing to me. It's also a very real probability that I could setup an automated storefront and not touch it for a year or two. Is that property still unused?

I think when it comes to this type of issue, it's something that people will have to work together with their neighbors, especially regarding aestetics.  I think that ultimately there will be virtual wastelands and junkyards. I also think there will be some wonderful areas and neighborhoods.  The community as a whole will form organically, most likely through land trading. This doesn't particularly address user drop out, but I say, it's their land, they've purchased it, and if they choose to abandon it that's their option. We will just have to work around it.

**Comment by pookage** [comment]
My colleague proposed something decent that I feel I should mention here - we shouldn't only be taking into account the activity on a plot of LAND by the owner, but also the amount that the land is visited. For example if someone built a sculpture and left it then that plot shouldn't be considered abandoned for as long as people are visiting it. 

I think that my opinion have shifted away from redistribution of abandoned LAND, and more towards temporarily repurposed. After x amount of being abandoned both by the owner and by visitors, the land would be loaned as 'public' land for a park or infrastructure - following attempts to contact the owner directly. As soon as the owner reclaims it then the land returns to its previous state and all carries on...

**Comment by rgiesick** [comment]
If I want to simply buy a plot of land and hold it forever isn't that my prerogative to do? After all I have paid for the ability to do so, there is no reason I must develop the land to generate income.

I plan on buying several plots of land and keep much of them mostly vacant, I want to put some distance between my plots and the noise of the plots around me, I want to establish my own aesthetic for the land I own. 

On the other hand part of claiming land is the chance that that land can increase in value, if you reclaim land from a person because that land has seemingly fallen out of use opens up real issues without compensation. let alone allowing a troll army to down vote them into oblivion just because you don't like what they are using their land for.

I understand that there is an issue with not wanting the world to turn into a wasteland, but not only can I sell my land and move to a more populated area but everyone else can as well.

**Comment by 3anz** [comment]
Let's introduce [Harberger LAND tax](https://medium.com/@simondlr/what-is-harberger-tax-where-does-the-blockchain-fit-in-1329046922c6). The idea is simple. Periodically, LAND will be auctioned and the highest bidder takes it after they pay the tax and previous owner. The period length, tax rate, and auction duration shall be decided by Decentraland DAO.

Here is a version of it:
When a LAND is up for auction, the LAND owner bids on their LAND like anyone else but needing to lock up only the tax amount of their bid. Others will lock up the full bid amount.

Only three cases can happen:
- **Case one**: No one outbids the LAND owner. The LAND owner keeps their LAND and their locked up MANA gets burned. Other bidders get back their locked up MANA.
- **Case two**: Someone outbids the LAND owner. The auction contract burns the LAND tax amount from the highest bid and sends the remaining MANA to the previous LAND owner plus the amount they had stacked. Other bidders get back their locked up MANA.
- **Case three**: No one bids. Owner retains their LAND.
